drivers/audio/tas2563.h

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2023, ithinx GmbH
+ * Copyright (c) 2023, tonies SE
+ *
+ * SPDX-License-Identifier: LicenseRef-Proprietary
```

Comment:
Please remove automatically assigned reviewers as long as you have work in progress like this and are still rapid-fire pushing.

PRs initially created as drafts have no reviewers but demoting to draft does not remove reviewers (too late!); they must be removed manually.

Reply:
sorry, I cannot remove the reviewers.

Reply:
However, I am done, it is ready for review.

Reply:
> Please remove automatically assigned reviewers as long as you have work in progress like this and are still rapid-fire pushing.
> 
> PRs initially created as drafts have no reviewers but demoting to draft does not remove reviewers (too late!); they must be removed manually.

@marc-hb is it OK to assign this PR to you?

Reply:
> However, I am done, it is ready for review.

There are still empty commit messages and failures in CI so it is not completely ready.

More importantly: don't mix changes independent from each other in the same PR, use separate PRs This is bad for reviewers and it's also bad for the submitter because the code does not get reviewed:

https://docs.zephyrproject.org/latest/contribute/contributor_expectations.html#defining-smaller-prs


> @marc-hb is it OK to assign this PR to you?

The generic parts should be reviewed by people from Intel. I don't know anything about TAS2563.